lib/Operation.php

```diff
@@ -117,6 +129,24 @@ public function isTaggingPath(IStorage $storage, string $file): bool {
 			return strpos($file, '__groupfolders') !== 0;
 		}
 
+		if (!$storage->isLocal()) {
```

Comment:
Doesn't work with local external storage.
But works fine with SFTP on 127.0.0.1

Reply:
added a commit that also takes local external storages into account